config/locales/devise/en.yml

```diff
@@ -85,18 +85,18 @@ en:
         email address. Please check your email and follow the confirm link to
         confirm your new email address.
       phone_update_needs_confirmation: >
-        Your request to update your phone number was processed successfully,
```

Comment:
I guess what I mean is, if we wanted it to be "snug" I would have expected you to bring up "but we need" or a few words from the next line onto this line